README.md

```diff
@@ -1 +1,99 @@
-# Capstone
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# Digital World 
+- In this Project I have created a site for an Digital World company, Financial based on the design of creative commons global summit 2015 designed by Cindy Shin in Behance. Here I created only two pages Home page and About Page using HTML, CSS and Javascript.
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+## Key Features
+- [Setting up of Linters]
+- [Adding Index.html and style.css]
+- [styling a text using class giving a appropriate font size,margin,image,padding]
+
+## Live Demo
+
+[Live Demo]()
+
```

Comment:
- [ ]  Your project should be hosted on GitHub pages properly and working as expected. but seemed that you didn't pass the link here. Please host your project and share the link in The Readme documentation.